src/app/common/components/ErrorModal.tsx

```diff
@@ -57,21 +57,37 @@ const ErrorModal: React.FunctionComponent<IProps> = (props) => {
             <>
               <br />
               <div>
-                A certificate error has occurred, likely caused by using self-signed CA certificates
-                in one of the clusters. Navigate to the following URL and accept the certificate:
+                A network error has occurred and we are unable to fetch one or more resources. We
+                are unable to determine the precise cause due to browser obfuscation, but based on
+                the content of the error it is likely one of two root issues:
```

Comment:
I am not sure the use will understand "browser obfuscation" or whether that information would be helpful. You might want to describe possible courses of action without explaining why the cause of the error is unknown.

Suggestion:

A network error has occurred. The likely cause of the error is one of the following issues: